### DISCUSSION TITLE
A solution to the "first move" problem

### DISCUSSION THREAD

**Discussion by ethanblake4** [PullDating/Org#80] [Ideas & Features]
This is pretty simple, but Bumble did get famous off of forcing women to make the first move. IMO not a super elegant solution. Why not have it so that if you swipe and match you're forced to make the first move before you can continue matching? That way it'd be equal between men and women, would make swipes more intentional, but less up-front effort than something like Hinge. I consider this kind of a 'buzzworthy' feature too: "Pull Dating is a new dating app that says it solves making the first move"

**Comment by WilliamDormer** [comment]
I really like the idea of forcing/encouraging the person whose swipe "activated" the match to be the one to send the first message. Solves the problem of people matching and just not sending a message. They would have to send the message as soon as they match, or else they lose it.

**Comment by Oshyan** [comment]
This is an interesting idea, but this sentence gives me pause:
>  you're forced to make the first move before you can continue matching

I have like 50 people in my Hinge Like (potential match) queue because I'm hesitant to make the call yes/no on those people. 😂 I'm not sure how to solve that, or if your suggestion is worse than the Hinge reality, but it's worth contemplating.

**Reply by ethanblake4** [reply]
I think that's kind of a separate issue. We haven't decided yet in what, if any way we'll allow people to see likes from other users. Probably lean towards no.

**Reply by Oshyan** [reply]
Really? Why? This is the first I've heard of limiting user access to basic info and, I'm open to debate on it to be sure, but my initial stance is pretty strongly against any limitations on user knowledge, empowerment, etc. In a certain sense, I might start with the idea that the line of what one user should know should stop only at where it invades the privacy of another or in some way actively harms others. And then perhaps extend that to also say that, if we have *good* reason to believe some particular limit will actually improve people's chances of finding what they're looking for, then those limits should also be seriously considered. 

Much of the discussion to-date, especially in Discord, has seemed to me to echo similar ideas, albeit not stated as strongly. Part of what existing commercial dating apps do *is* impose limits for their own commercial reasons. If we have good, user-oriented, pro-social reasons to limit info, then great. But again my default stance against any such things is no, heck no.

**Reply by ethanblake4** [reply]
I mean I [have nothing against it](https://github.com/PullDating/Org/discussions/77), but based on my conversation with Will it didn't seem like that's the direction he wanted to go. Personally I think there's merit to both approaches in terms of what's best for the user. For one, letting people see likes would (as you pointed out) conflict with this proposal which I do quite like. I also don't want to straight up copy Hinge because then why would you use this app? On the other hand it is probably the app that's closest to our goals, so idk.

**Comment by rjheby** [comment]
Could we ease pressure of the first move for shyer partners by allowing them to choose a topic instead of writing a message. E.g. they swipe right, BOOM Match. They can say whatever they want or leave a little message that says "talk to me about... (Movies, TV, Food)"

**Comment by shootingcharlie8** [comment]
I really like this idea. My input on this would have a minimum character count of say... 25?

**Comment by ajbott** [comment]
Really good community control in terms of the 1st match.
